monorepo-folders/README.md

```diff
@@ -0,0 +1,65 @@
+# monorepo-folders
+
+This example is meant to help people with alternative folder structures to our hello-world:
+
+```bash
+packages/
+  backend-apis/
+    server.ts # runs Express server on localhost:4000, has Temporal Client that calls workflows
+  frontend-ui # this is a Create-React-App on localhost:3000, proxies api requests to :4000
+  temporal-worker/
+    worker.ts # registers Temporal Worker that has Workflows and Activities from /temporal-workflows
+  temporal-workflows/
+    lib/ # compiled workflows
+    src/
+      workflowA/
+          activities/
+            activitiesA.ts
+            activitiesB.ts
+          workflow.ts
+      workflowB/
+          activities/
+            activitiesC.ts
+            activitiesD.ts
+          workflow.ts
+      workflowE.ts
+      all_workflows.ts
+      activityF.ts
+node_modules
+package.json
+```
+
+Notes on the structure demonstrated:
+
+- **Workflows require one file**: you can organize workflows however you like, but you need to export them all in a single file per Worker (so that the Worker's webpack has an entry point)
```

Comment:
Did we talk about having `workflowsPaths` plural? Don't see it in the current API. If not, I'd say:

`**One Workflow file per Worker**`: you can organize Workflow code however you like, but each Worker needs to reference a single file that exports all the Workflows it handles (so that the Worker's webpack has an entry point)

Does a Worker need to be able to handle all Workflows on the queue? If yes, can change "Workflows it handles" to "Workflows executed on its task queue".

Reply:
Could maybe take this opportunity to show two additional workers—one that only knows about workflowA and activitesA/B, and the other that knows workflowB and activitiesC/D? Or could be another sample

Reply:
- no i dont think we discussed plural wfpaths - i think @bergundy had a reason for not doing it
- no, worker listening to queue without the right workflow type registered will simply not pick it up
- i think its ok to show additional workers yeah, will probably do that here rather than adding yet another sample